apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -55,7 +55,7 @@ class ProgressLessonTeacherInfo extends React.Component {
 
   onClickHiddenToggle(value) {
     const {scriptName, section, lesson, toggleHiddenStage} = this.props;
-    const sectionId = (section && section.id.toString()) || '';
+    const sectionId = (section && section.id) || null;
```

Comment:
imo would be preferable to use `teacherSectionsRedux.NO_SECTION` here and at line 91

Reply:
I ended up removing this null altogether and adding a comment about why sectionId should never be null here.  I left the one on line 91 as-is because I thought null was a more appropriate value given how it is used later in the stack (used with a "if" check to determine if a section was present, as a proxy for figuring out if this is a student).  I hope we'll have time to revisit this in H1.  The logic here all seems more complicated than it needs to be, in part because of how teacher and student code are intermingled.